### PR TITLE
#114

### PR DESCRIPTION
**Pull request by gzinck**
Fixed and added testing for the initial state, initial load, and logout functionality in the redux store.

Note that some functions were exported that used to be private. This was necessary in order to test the functions.

The testing uses **Jest** because that was the testing framework already listed in our `package.json`. Since I wanted to test this (but I didn't want to install anything new), I just used what we already had. It turns out that Jest is quite good, and the Redux docs have some guidance on how to test things in the redux store using it!

[https://jestjs.io/docs/en/tutorial-react](url)

### CODE REVIEW

**Comment by evscott** [comment]
> Fixed and added testing for the initial state, initial load, and logout functionality in the redux store.
> 
> Note that some functions were exported that used to be private. This was necessary in order to test the functions.
> 
> The testing uses **Jest** because that was the testing framework already listed in our `package.json`. Since I wanted to test this (but I didn't want to install anything new), I just used what we already had. It turns out that Jest is quite good, and the Redux docs have some guidance on how to test things in the redux store using it!
> 
> [https://jestjs.io/docs/en/tutorial-react](url)

I will give this pull request a thorough review in a little bit, but Jest is a great testing framework! I'd be fine if we continued using it moving forward. It would be worth discussing this with the group however, since Morgan has found a possible alternative.

File: src/redux/actions/authActions.js [hunk 1]
```diff
@@ -77,12 +77,26 @@ function logoutFailure() {
...
     isAuthenticated: false,
-    user: null
+    isFetching: false,
+    isSynced: true,
+    user: [],
+    accoms: [],
+    plans: [],
+    trans: [],
+    trips: []
   };
```
Comment (evscott) [inline src/redux/actions/authActions.js]: This looks good. Though, ` isSynced: true` should be ` isSynced: false`.

File: src/redux/reducers/accomReducer.js [hunk 2]
```diff
@@ -11,6 +12,14 @@ const initialState = {
...
 export function accomReducer(state = initialState, action) {
   switch (action.type) {
+    case LOGOUT_SUCCESS:
+      return {
+        ...state,
+        lastUpdated: action.lastUpdated,
+        isFetching: action.isFetching,
+        isSynced: action.isSynced,
+        accoms: action.accoms
+      };
```
Comment (evscott) [inline src/redux/reducers/accomReducer.js]: Ahaa! I see how you've done this now. Clever!

File: src/redux/actions/authActions.js (hunk 1 above)
Comment (gzinck) [reply src/redux/actions/authActions.js]: Good point! Changed it just now.

**Pull request merged**